ftml/src/preproc/typography.rs

```diff
@@ -189,14 +196,41 @@ const TEST_CASES: [(&str, &str); 3] = [
         "**ENTITY MAKES DRAMATIC MOTION** . . . ",
         "**ENTITY MAKES DRAMATIC MOTION** … ",
     ),
+    ("Whales... they are cool", "Whales… they are cool"),
+    ("Whales ... they are cool", "Whales … they are cool"),
+    ("...why would you think that?", "…why would you think that?"),
+    ("how could you...", "how could you…"),
+    (
+        "... why would you think that?",
+        "… why would you think that?",
+    ),
+    ("how could you ...", "how could you …"),
+    ("Whales. . . they are cool", "Whales… they are cool"),
+    (
+        ". . .why would you think that?",
+        "…why would you think that?",
+    ),
+    ("how could you. . .", "how could you…"),
+    ("Whales . . . they are cool", "Whales … they are cool"),
+    (
+        ". . . why would you think that?",
+        "… why would you think that?",
+    ),
+    ("how could you . . .", "how could you …"),
```

Comment:
I think this test structure is a bit strange. It looks like there are three classes of test, prefix, suffix, and infix (i.e. whether the `...`s are at the start, end, or in the middle), and that you test each of `...` and `. . .`. I would suggest that they have a consistent order, for instance:
```rust
"Whales... they are cool",
"Whales. . . they are cool",
"...why would you think that?",
". . .why would you think that?",
"how could you ...",
"how could you . . .",
```

Reply:
They have now been reordered, is it now?